### ISSUE TITLE
Can we eliminate the test_might_fail in the last test of t/0700-unifyfs-stage-full.t

### ISSUE THREAD

**Issue by craigsteffen** [LLNL/UnifyFS#616]
### System information

Type                   | Version/Name
---                    |     ---
Operating System       | all
OS Version             |
Architecture           |
UnifyFS Version        |

### Describe the problem you're observing

The last test in the file t/0700-unifyfs-stage-full.t has the test tag: 
test_might_fail 
This shouldn't be necessary.  We think it was put in at some point because Travis was tripping over that test.  The test normally works correctly on real systems.

**Comment by craigsteffen** [comment]
Craig will create a PR to see if this is still necessary.  We can test the Travis testing on this by removing the tag and seeing if the sharness allows it to go through.

**Comment by CamStan** [comment]
I agree that we should remove the `test_might_fail`. Go ahead with that PR as it would be good to know if this is failing on Travis.
However, @jenest is also seeing this fail on Summit, so some debugging will likely be needed. It does pass for me when I remove it and run on LLNL systems.

**Comment by craigsteffen** [comment]
Jenna and I tested this yesterday.  It seems to work fine with or without the test_might_fail tag on the last test in the 0700 script.   

She's going to finish up the refactoring PR that she's working on for issue 516.  Then from that branch, she's going to create a branch and a PR to make this change (remove the tag from 0700).  

I'm available to assist, but we decided it would probably be simpler if these changes were made in one set of branches under one person, rather than us trying to hand branches back and forth.
